src/main/java/lotto/model/WinnerCollection.java

```diff
@@ -0,0 +1,25 @@
+package lotto.model;
+
+import java.util.List;
+
+public class WinnerCollection {
```

Comment:
일급 컬렉션을 잘 만들어주셨네요!
약간의 첨언을 하자면, 
```suggestion
public class Winners {
```
로 충분한 것 같습니다. Collection이라는 구조를 노출하지 않아도 될 것 같아요:)

그리고 개인적으로는...Winner라는 네이밍이 적절한지 잘 모르겠습니다. Winner라면 왠지 사람?을 뜻하는거같아서요...이건 그냥 개인적인 생각입니다!!